openssh/README.md

```diff
@@ -0,0 +1,132 @@
+hab pkg install -b chef/inspec
+
+# install the openssh artifact
+hab pkg install -b results/$pkg_artifact
+
+# restart the habitat supervisor and wait for it to start
+hab sup run &
+sleep 20
+
+# load the openssh habitat service
+hab svc load $pkg_ident
+set +x
+```
+
+Wait until the service has loaded successfully.  At this stage something like the following will appear in the terminal:
+
+```bash
+openssh.default(CF): Created configuration file /hab/svc/openssh/config/sshd_config
+openssh.default(SR): Initializing
+openssh.default hook[init]:(HK): Generating public/private dsa key pair.
+openssh.default hook[init]:(HK): Your identification has been saved in /hab/svc/openssh/config/ssh_host_dsa_key.
+openssh.default hook[init]:(HK): Your public key has been saved in /hab/svc/openssh/config/ssh_host_dsa_key.pub.
+...
+...
+openssh.default hook[init]:(HK): The key's randomart image is:
+openssh.default hook[init]:(HK): +---[ECDSA 256]---+
+openssh.default hook[init]:(HK): |*^O== .          |
+openssh.default hook[init]:(HK): |%.*= *           |
+openssh.default hook[init]:(HK): |+Bo.= .          |
+openssh.default hook[init]:(HK): |o=++ o .         |
+openssh.default hook[init]:(HK): |o.+ o + S        |
+openssh.default hook[init]:(HK): |oo E o           |
+openssh.default hook[init]:(HK): |+..              |
+openssh.default hook[init]:(HK): |o..              |
+openssh.default hook[init]:(HK): |=+.              |
+openssh.default hook[init]:(HK): +----[SHA256]-----+
+openssh.default hook[init]:(HK): + '[' '!' -f /hab/svc/openssh/config/ssh_host_dsa_key ']'
+openssh.default hook[init]:(HK): + ssh-keygen -t dsa -f /hab/svc/openssh/config/ssh_host_dsa_key -N ''
+openssh.default hook[init]:(HK): + '[' '!' -f /hab/svc/openssh/config/ssh_host_rsa_key ']'
+openssh.default hook[init]:(HK): + ssh-keygen -t rsa -f /hab/svc/openssh/config/ssh_host_rsa_key -N ''
+openssh.default hook[init]:(HK): + '[' '!' -f /hab/svc/openssh/config/ssh_host_ed25519_key ']'
+openssh.default hook[init]:(HK): + ssh-keygen -t ed25519 -f /hab/svc/openssh/config/ssh_host_ed25519_key -N ''
+openssh.default hook[init]:(HK): + '[' '!' -f /hab/svc/openssh/config/ssh_host_ecdsa_key ']'
+openssh.default hook[init]:(HK): + ssh-keygen -t ecdsa -f /hab/svc/openssh/config/ssh_host_ecdsa_key -N ''
+openssh.default hook[init]:(HK): + set +x
+openssh.default(SV): Starting service as user=root, group=root
+openssh.default(O): + exec /hab/pkgs/bigbird/openssh/7.5p1/20190326193307/sbin/sshd -D -f /hab/svc/openssh/config/sshd_config
+```
+
+Remain in the same habitat studio for the next section
+
+### Run the tests
+
+```bash
+hab pkg exec chef/inspec inspec exec ./test
+```
+
+Sample output:
+
+```bash
+[7][default:/src:0]# hab pkg exec chef/inspec inspec exec ./test
+
+Profile: Habitat Core Plan openssh (openssh)
+Version: 0.1.0
+Target:  local://
+
+  ✔  ensure openssh binary service works: Bash command /hab/pkgs/bigbird/openssh/7.5p1/20190326193307/sbin/sshd  --help 2>&1
```

Comment:
Suggest changing `bigbird` to `core` as these tests are for directly testing the core plans here in this repo.